pkg/api/rest/rest.go

```diff
@@ -107,6 +107,13 @@ func (w GracefulDeleteAdapter) Delete(ctx api.Context, name string, options *api
 	return w.Deleter.Delete(ctx, name)
 }
 
+// An object that can delete a collection of RESTful resources in a namespace
```

Comment:
Just say "An object can that can delete a collection of RESTful resources"

I could see this being implemented for non-namespaced resources, for example, to delete all nodes.